### PR TITLE
Update /actions page #75 (Partial)

### PR DESCRIPTION
**Pull request by MazharulIslam-Naim**
## Overview 
Fixed the navigation bar (navigating to the correct pages when clicking on "Take Actions")
Added crowns to challenger emoji when player completes actions
Updated (show/remove) buttons based on completed actions
Added “you are done” page for when a player completes all actions 
Added “see your challenge” link for challengers on this page

## Test Plan 
This was tested manually. To test this you can comment out the state set(red) and change the values manually(green). 
image

## Follow ups 
Sharing about actions completed for challenger is not done. I am still working on that.

Instead of doing a separate page that the user gets forwarded to, the actions page changes appearance based on if they completed all actions. Also, I saw that there was a separate page mocked for when only taking the challenge action is left, so that's what appears when the user only has Take Challenge action left.

### CODE REVIEW

File: src/components/Actions/Actions.js [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+  const history = useHistory();
+  const [loading, setLoading] = useState(true);
+  const [challengerInfo, setChallengerInfo] = useState(null);
+  const [ registeredVoter, setRegisteredVoter ] = useState(false);
+  const [ notifyElectionReminders, setNotifyElectionReminders ] = useState(false);
+  const [ startedChallenge, setStartedChallenge ] = useState(false);
+
+  useEffect(() => {
+    if (localStorage.getItem("challengerInfo")) {
+      setChallengerInfo(JSON.parse(localStorage.getItem("challengerInfo")));
```
Comment (Devanshusp) [inline src/components/Actions/Actions.js:31]: We may need to add a backup here for when there is no challenger info in the local storage (ex. user uses a different device)

**Review by Devanshusp — APPROVED** [review]
Thank for working on the actions page @MazharulIslam-Naim, this looks amazing!
The variations work great on the actions page and everything else works as expected 😄! 

Looking forward to the sharing card, I believe that is the only design left on this page

**Comment by MazharulIslam-Naim** [comment]
Hey, @Devanshusp I added what you said and the sharing pop-up, take a look and see if there is anything wrong.

**Comment by MazharulIslam-Naim** [comment]
Actually I found two bugs with this flow so I'll fix those and push that before this gets merged.

**Comment by MazharulIslam-Naim** [comment]
If someone wants to review the last three commits I made on this pr and everything looks ok, then this can be merged.

**Comment by rickyyx** [comment]
Hey @MazharulIslam-Naim great work here! Really love how clean and well documented the code is. This is really high-quality. 

One quick comment on the sharing of action, should we be sharing the original challenger's challenge here? We might have to extend/refactor the sharing code generation function so that it could differentiate different user (player vs challenger). For example, if I am a player, and when I share my action for Yang's challenge as shown in the design below, I should be generating links/sharing content based on Yang rather than me I guess? 

image

**Comment by MazharulIslam-Naim** [comment]
Hey, @githubxxcc thanks. Ya I'm currently working on this right now, that will be a separate pr. I put in the invite component there because I'm working of that component; it's just temporary so you can see how it will be.

**Pull request merged**